packages/shared/src/guards.ts

```diff
@@ -458,12 +471,6 @@ export function isNdArray<T extends Data>(
   return 'data' in arr;
 }
 
-export function isTypedArray<T extends TypedArray>(
-  arr: T | unknown[],
-): arr is T {
-  return !Array.isArray(arr);
-}
-
```

Comment:
This function was confusingly named. Also, it's so simple that it really doesn't warrant its own type guard.